### PR TITLE
Build Wihtout fastapi

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

ClobberError: This transaction has incompatible packages due to a shared path.
  packages: conda-forge/noarch::fastapi-0.113.0-pyhd8ed1ab_0, conda-forge/noarch::fastapi-cli-0.0.5-pyhd8ed1ab_0
  path: 'bin/fastapi'

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,15 +24,13 @@ requirements:
...
-    - fastapi
-    - uvicorn >=0.15.0
+    - uvicorn-standard >=0.15.0
 
 test:
   imports:
     - fastapi_cli
   commands:
     - pip check
-    - fastapi --help
```
Comment (synapticarbors) [inline recipe/meta.yaml:35]: If the upstream `pyproject.toml` defines the entry point (https://github.com/fastapi/fastapi-cli/blob/main/pyproject.toml#L52), I think we should probably test it, unless there is a reason not to.

**Review by synapticarbors — COMMENTED** [review]
Thanks for the PR. I think it looks correct and just have one question about removing one of the `command` tests.

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Pull request merged**